app/components/file-uploader.js

```diff
@@ -131,7 +154,7 @@ export default Ember.Component.extend(Analytics, {
         createProjectAndUploadFile() {
             // Upload case where user starting from scratch - new project/new file.  Creates project and then uploads file to newly
             // created project
-            Ember.get(this, 'metrics')
+            get(this, 'metrics')
```

Comment:
Is there a reason we are doing `get(this, 'metrics')` instead of `this.get('metrics')` everywhere? In v3.1 of ember we can start doing `this.metrics` so it might be best to go ahead an switch to `this.get('metrics')`.